test/contrib/s3_test.py

```diff
@@ -235,7 +235,7 @@ def test_put_multipart_less_than_split_size(self):
         # 5MB is minimum part size
         part_size = 8388608
         file_size = 5000
-        self._run_multipart_test(part_size, file_size)
+        return self._run_multipart_test(part_size, file_size)
```

Comment:
Why do you return the last thing you do in a test case??? I thought you only used self.assertBlah()

Reply:
The `test_put_multipart...` tests are used below as args to `self._run_copy_test(...)` and I need the output of `self._run_multipart_test` to verify if the correct num and size of files were copied